csrc/scheduler/greedy.cpp

```diff
@@ -837,6 +941,11 @@ void insertCopies(Fusion* fusion, GreedyParams& greedy_params) {
           // Insert an exclusive copy
           auto inp_copy = set(inp_tv);
           expr = ir_utils::replaceValInExprInputs(expr, inp_tv, inp_copy);
+          greedy_params.transferProducerParams(
```

Comment:
nitpick: I know we don't support TopKOp with batching. so it doesn't matter. Maybe throw a TODO that TopKOp has multiple outputs, so when batching is supported we'll need to extend this.